addons/greatsql-cluster/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+annotations:
```

Comment:
pls add more annotations to speicify its dependenc on KB version, such as 
```yaml
  addon.kubeblocks.io/kubeblocks-version: ">=0.8.0"
  addon.kubeblocks.io/model: "RDBMS"
  addon.kubeblocks.io/provider: "community"
```

Reply:
The annotations of  its dependencies on the KB version are in greatsql's chart.yaml file, not in greatsql-cluster's chart.yaml file. All the other addons in Kubeblock-addons are like this.